lib/data/wrapper.js

```diff
@@ -207,7 +212,7 @@ const data = {
                 message: result.statusMessage,
             };
             return cb(null, respBody);
-        });
+        }, deep);
```

Comment:
please switch the order of the arguments so you don't have to have an argument after the callback.